ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -384,11 +381,12 @@ export default defineComponent({
             localState.ownerGroupId
           )
         }
-        setGroupFractionalInterest(
-          localState.ownerGroupId || '1',
-          localState.fractionalData
-        )
-        localState.ownerGroupId && setShowGroups(true)
+
+        setShowGroups(Number(localState.ownerGroupId) > 0)
+
+        // Get fractional data based on owner's group id
+        const fractionalData = find(allFractionalData, { groupId: localState.ownerGroupId })
```

Comment:
Maybe a type here would be helpful. I imagine you already have an interface for these data structures?

Reply:
Yes, I do. Will add.

Reply:
Added interface.

Reply:
Looks good.  It will be helpful.